scripts/alert-check.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+# USAGE
+# ./alerts-during-perf-testing.sh <optional sleep in seconds> <optional product-name>
+# ^C to break
+# Generates two files one for firing and one for pending alerts
+#
+# PREREQUISITES
+# - jq
+# - oc (logged in at the cmd line in order to get the bearer token)
+# VARIABLES
+
+# wait for RHMI CR to appear and have namespace prefix populated
+until oc get RHMIs --all-namespaces -o jsonpath='{.items[0].spec.namespacePrefix}' &> /dev/null
+do
+    echo "Waiting for RHMI CR with namespace prefix set to appear on cluster. Next check to be in 1 minute."
+    sleep 60
+done
+
+NAMESPACE_PREFIX="${NAMESPACE_PREFIX:-$(oc get RHMIs --all-namespaces -o jsonpath='{.items[0].spec.namespacePrefix}')}"
+
+# wait for monitoring route to appear and have host populated
+until oc get route prometheus -n ${NAMESPACE_PREFIX}observability -o=jsonpath='{.spec.host}' &> /dev/null
+do
+    echo "Waiting for Prometheus route in ${NAMESPACE_PREFIX}observability namespace to appear on cluster. Next check to be in 1 minute."
+    sleep 60
+done
+
+SLEEP_TIME="${1-5}"
+RHSSO="rhsso"
+USER_SSO="user-sso"
+ROUTE="http://localhost:9090/api/v1/alerts"
+THREESCALE="3scale"
```

Comment:
Dont think we need these.

```suggestion
ROUTE="http://localhost:9090/api/v1/alerts"
```